examples/config.rs

```diff
@@ -0,0 +1,35 @@
+extern crate cursive;
+extern crate cursive_hexview;
+
+use cursive::views::{Dialog, DummyView, LinearLayout, TextView};
+use cursive_hexview::{DisplayState, HexView, HexViewConfig};
+
+fn main() {
+    let mut cur = cursive::default();
+    let explanation = TextView::new("Use the keys + - ↑ ↓ ← → 0-9 a-f for the HexView.\nUse q to exit.");
+    let data = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789".as_bytes();
+    let view1 = HexView::new_from_iter(data).display_state(DisplayState::Editable);
+    let mut config = HexViewConfig {
+        hex_ascii_separator : " - ".to_owned(),
+        ..Default::default()
+    };
+
+    config.bytes_per_line = 8;
+    config.bytes_per_group = 4;
+    config.chars_per_spacing = 5;
```

Comment:
any reason why those aren't merged? Let's merge them, looks better (and people who use rust should know how to mutate a struct with pub fields ;) )

Reply:
The point I wanted to make was actually that you can use Default instead of having to change all, but as you said, maybe people just knows that stuff. I'm new to Rust so I thought it was useful to exemplify it like that. I've changed it now in the example file as part of [the change to have a non chainable version](https://github.com/palako/cursive_hexview/commit/1539f7e76ad27b4c684b7fe2677bb2c16ad99b08):